boxoffice/models/line_item.py

```diff
@@ -152,6 +156,18 @@ def confirm(self):
     def current_assignee(self):
         return self.assignees.filter(Assignee.current == True).one_or_none()  # NOQA
 
+    @property
+    def is_assignable(self):
+        return utcnow().date() <= self.item.event_date
+
+    @property
+    def is_transferable(self):
+        if self.current_assignee is None:
+            return self.is_assignable
+        else:
+            return utcnow() <= self.item.transferable_until if self.item.transferable_until is not None else self.is_assignable
```

Comment:
Should be `< self.item.transferable_until`. If you specify the transfer cut-off as midnight, then it should work until midnight but not at midnight.